recipes/argparse/config.yml

```diff
@@ -1,3 +1,3 @@
 versions:
-  "2.1":
+  "2.2":
```

Comment:
Same remark here as in conandata.yml:
don't remove the previous version but add it.